releasenotes/notes/fix-digraph-find-cycle-141e302ff4a8fcd4.yaml

```diff
@@ -0,0 +1,8 @@
+---
+fixes:
```

Comment:
I think we need an upgrade note about the trait bound changes to the public rustworkx-core function.

Reply:
Done